frontend/console/e2e/cron.spec.ts

```diff
@@ -28,6 +29,7 @@ test('send cron request', async ({ page }) => {
 test('submit cron form using ⌘+⏎ shortcut', async ({ page }) => {
   await navigateToDecl(page, 'cron', 'thirtySeconds')
 
+  await setVerbRequestBody(page, '{}')
```

Comment:
@stuartwdouglas I think this is why the test was failing before. This bug probably existed before your PR btw, we just never hit the failure case here. Basically it was pressing the keyboard shortcut before the page had fully loaded and the request body was available. This should fix that.